apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -147,6 +146,23 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
                .addShutdownHook(new Thread(ServiceManager.INSTANCE::shutdown, "skywalking service shutdown thread"));
     }
 
+    private static AgentBuilder newAgentBuilder(Instrumentation instrumentation, String nameTrait) {
```

Comment:
```suggestion
        /**
     * Create a new agent builder through customized {@link ByteBuddy} powered by 
     * {@link SWAuxiliaryTypeNamingStrategy} {@link DelegateNamingResolver} {@link SWMethodNameTransformer} and {@link SWImplementationContextFactory}
     */
    private static AgentBuilder newAgentBuilder() {
```

You actually don't need all these parameters and recommend adding comments like this to indicate the changes.